src/ProjectOrigin.Vault/CommandHandlers/VerifySliceCommandHandler.cs

```diff
@@ -127,7 +127,8 @@ private async Task InsertIntoWallet(VerifySliceCommand receivedSlice, GranularCe
                 EndDate = registryCertificateProjection.Period.End.ToDateTimeOffset(),
                 GridArea = registryCertificateProjection.GridArea,
                 CertificateType = (GranularCertificateType)registryCertificateProjection.Type,
-                Attributes = attributes
+                Attributes = attributes,
+                Withdrawn = false
```

Comment:
Is this the real logic? Or is this a todo for next task?

Reply:
I'm not sure what you mean? But on another node, I need to filter withdrawn certificates

Reply:
Withdrawn = false <-- This part. Hardcoded. Shouldn't it be Withdrawn = withdrawn;?

Reply:
nicely spotted. corrected